spec/system/admin/products_v3/products_spec.rb

```diff
@@ -28,13 +19,28 @@
   end
 
   it "can see the new product page" do
-    visit "/admin/products_v3"
+    visit admin_products_v3_index_url
     expect(page).to have_content "Bulk Edit Products"
   end
 
-  context "pagination" do
-    before :each do
-      visit "/admin/products_v3"
+  describe "sorting" do
+    let!(:product_z) { create(:simple_product, name: "zucchini") }
+    let!(:product_a) { create(:simple_product, name: "Apples") }
+
+    before do
+      visit admin_products_v3_index_url
+    end
+
+    it "Should sort products alphabetically by default" do
+      expect(page).to have_selector "table.products tbody tr td", text: "Apples"
+      # other products push later one to next page
+      expect(page).not_to have_selector "table.products tbody tr td", text: "zucchini"
```

Comment:
That's a good point, because if the product wasn't showing at all, the test would still pass!

But this test doesn't need to be about pagination, so an even better test would be that "Apples" appears before "Bananas" on the same page. Hmm I will try that.